test/e2e/network/service.go

```diff
@@ -3961,6 +3961,7 @@ var _ = SIGDescribe("SCTP [Feature:SCTP] [LinuxOnly]", func() {
 	})
 
 	ginkgo.It("should create a Pod with SCTP HostPort", func() {
+		e2eskipper.RunIfContainerRuntimeIs("docker")
 		ginkgo.By("checking whether kubenet is used")
```

Comment:
can someone explain why this is checking for kubenet?

Reply:
The obvious way to test "SCTP HostPorts work" would be to create a pod with an SCTP HostPort, and then try to send traffic to it. But we can't do that because many hosts (including all GCE-based nodes) won't have the SCTP kernel modules so trying to actually send SCTP traffic would fail. So instead we check that it's working indirectly, by seeing that the expected iptables rules get created. But that only works if we're sure we know what iptables rules will be created, so we have to whitelist where the test can run

Reply:
in the linked issues there's discussion of not loading the module so as to allow usage of userspace SCTP -- can we test that instead?

Reply:
In theory, yes, though we'd have to maintain some image with appropriate useful test binaries built against some userspace SCTP library, which would be a lot of work for a relatively marginal feature.

Note that there's a second set of SCTP tests, not yet complete, labeled `[Feature:SCTPConnectivity]` that actually check that SCTP connections work. But those tests can only be run in clusters that have the SCTP kernel module available.

Reply:
ACK -- at some point i'd like to follow up about what it would take to get the module loaded.
A lot of the CI is on GCE / Ubuntu nodes these days, it may be more viable than GCE / cos.

KIND will have the containerd patch today, we'd just need the module and a job that ran this test tag, the underyling VMs are GKE/Ubuntu but e.g. we load the IPv6 modules.